### PR TITLE
Created a delay option for the click event

### PR DESCRIPTION
**Pull request by Nevgup**
Hello

I make this PR following the problem I had here : https://github.com/Xetera/ghost-cursor/issues/24
I have a low level in Typescript, if I haven't done anything correct don't hesitate to let me know because it would allow me to improve.

### CODE REVIEW

**Comment by Nevgup** [comment]
You mean i should do that ?
`interface ClickOptions extends MoveOptions { readonly waitForClick: number, delayOptions: number }`

File: src/spoof.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export { default as installMouseHelper } from './mouse-helper'
 
 interface BoxOptions { readonly paddingPercentage: number }
 interface MoveOptions extends BoxOptions { readonly waitForSelector: number }
-interface ClickOptions extends MoveOptions { readonly waitForClick: number }
+interface ClickOptions extends MoveOptions { readonly waitForClick: number, readonly delayOptions: number }
```
[the remark this answers is not in the document]
Comment (Xetera) [reply src/spoof.ts:7]: It should probably also be called something more descriptive like `moveDelay`

**Comment by Nevgup** [comment]
Now it's good ?

**Review by Niek — CHANGES_REQUESTED** [review]
@Nevgup can you please check my pending reviews?

File: src/spoof.ts [hunk 2]
```diff
@@ -113,9 +113,9 @@ const getElementBox = async (page: Page, element: ElementHandle, relativeToMainF
   return elementBox
 }
 
-export function path (point: Vector, target: Vector, spreadOverride?: number)
-export function path (point: Vector, target: BoundingBox, spreadOverride?: number)
-export function path (start: Vector, end: BoundingBox | Vector, spreadOverride?: number): Vector[] {
+export function path(point: Vector, target: Vector, spreadOverride?: number)
+export function path(point: Vector, target: BoundingBox, spreadOverride?: number)
+export function path(start: Vector, end: BoundingBox | Vector, spreadOverride?: number): Vector[] {
```
[the remark this answers is not in the document]
Comment (Nevgup) [reply src/spoof.ts]: I think it's good !

**Pull request merged**